src/storage/v2/schemas.cpp

```diff
@@ -19,21 +19,39 @@
 namespace memgraph::storage {
 
 SchemaViolation::SchemaViolation(ValidationStatus status, LabelId label) : status{status}, label{label} {}
-SchemaViolation::SchemaViolation(ValidationStatus status, LabelId label, SchemaProperty violated_type)
+SchemaViolation::SchemaViolation(ValidationStatus status, LabelId label, SchemaPropertyType violated_type)
     : status{status}, label{label}, violated_type{violated_type} {}
 
-SchemaViolation::SchemaViolation(ValidationStatus status, LabelId label, SchemaProperty violated_type,
+SchemaViolation::SchemaViolation(ValidationStatus status, LabelId label, SchemaPropertyType violated_type,
                                  PropertyValue violated_property_value)
     : status{status}, label{label}, violated_type{violated_type}, violated_property_value{violated_property_value} {}
 
-bool Schemas::CreateSchema(const LabelId primary_label, const std::vector<SchemaProperty> &schemas_types) {
-  return schemas_.insert({primary_label, schemas_types}).second;
+Schemas::SchemasList Schemas::ListSchemas() const {
+  Schemas::SchemasList ret;
+  ret.reserve(schemas_.size());
+  for (const auto &[label_props, schema_property] : schemas_) {
+    ret.emplace_back(label_props, schema_property);
+  }
+  return ret;
 }
 
-bool Schemas::DeleteSchema(const LabelId primary_label) {
-  return schemas_.erase(primary_label);
+Schemas::SchemasList Schemas::GetSchema(const LabelId primary_label) const {
+  if (auto schema_map = schemas_.find(primary_label); schema_map != schemas_.end()) {
+    return {{schema_map->first, schema_map->second}};
+  }
+  return {};
+}
+
+bool Schemas::CreateSchema(const LabelId primary_label, const std::vector<SchemaPropertyType> &schemas_types) {
+  if (schemas_.contains(primary_label)) {
+    return false;
+  }
+  schemas_.insert({primary_label, schemas_types});
+  return true;
 }
 
+bool Schemas::DropSchema(const LabelId primary_label) { return schemas_.erase(primary_label); }
```

Comment:
Opted for Drop, since we use Create/Delete for data, and Create/Delete for indexes and constraints, it makes sense to use the same for schema